pkg/resource/unstructured/client.go

```diff
@@ -53,7 +52,7 @@ type WrapperClient struct {
 // Get retrieves an obj for the given object key from the Kubernetes Cluster.
```

Comment:
Newly added `Scheme()` and `RESTMapper()` are missing in `WrapperClient`. I believe adding something like this would work:

```go
// Scheme returns the scheme this client is using.
func (c *WrapperClient) Scheme() *runtime.Scheme {
	return c.kube.Scheme()
}

// RESTMapper returns the rest this client is using.
func (c *WrapperClient) RESTMapper() meta.RESTMapper {
	return c.kube.RESTMapper()
}
```

Reply:
Alright, added the two methods as suggested. :+1: